var/spack/repos/builtin/packages/dla-future-fortran/package.py

```diff
@@ -24,7 +24,8 @@ class DlaFutureFortran(CMakePackage):
     version("0.2.0", sha256="7fd3e1779c111b35f0d2701a024398b4f6e8dea4af523b6c8617d28c0b7ae61a")
     version("0.1.0", sha256="9fd8a105cbb2f3e1daf8a49910f98fce68ca0b954773dba98a91464cf2e7c1da")
 
-    depends_on("fortran", type="build")  # generated
+    depends_on("c", type="build")
+    depends_on("fortran", type="build")
```

Comment:
```suggestion
    depends_on("c", type="build")
    depends_on("cxx", type="build")
    depends_on("fortran", type="build")
```

In the `CMakeLists.txt` we currently set all three languages.

Reply:
@RMeli

Reply:
Thanks!